packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -450,7 +448,7 @@ const CFDsListing = observer(() => {
                                 onAction={(e?: React.MouseEvent<HTMLButtonElement>) => {
                                     const button_name = e?.currentTarget?.name;
                                     const track_account_subtitle = is_demo
-                                        ? `${existing_account.sub_title} ${localize('Demo')}`
+                                        ? `${existing_account.name} ${'Demo'}`
                                         : existing_account.sub_title;
```

Comment:
here `track_account_subtitle` is unused as it has been replaced with `track_account_name`